atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java

```diff
@@ -270,5 +270,43 @@ private void registerMetrics(GenericObjectPool<Client> pool, String metricPrefix
                 CassandraClientPoolingContainer.class,
                 metricPrefix, "proportionDestroyedByBorrower",
                 () -> ((double) pool.getDestroyedByBorrowValidationCount()) / ((double) pool.getCreatedCount()));
+
+        // Count of active or idle threads in the pool
+        metricsManager.registerMetric(
+                CassandraClientPoolingContainer.class,
+                metricPrefix, "numActive",
+                pool::getNumActive);
+        metricsManager.registerMetric(
+                CassandraClientPoolingContainer.class,
+                metricPrefix, "numIdle",
+                pool::getNumIdle);
+
+        // Proportion of active and idle threads in the pool
+        metricsManager.registerMetric(
+                CassandraClientPoolingContainer.class,
+                metricPrefix, "proportionActiveThreads",
+                () -> ((double) pool.getNumActive()) / ((double) pool.getCreatedCount()));
```

Comment:
might want getCreatedCount == 0.0  checking; could see that temporarily being the case in some just-starting-up edge cases, or when a user configures a stack to idle at zero connections rather than our default idle of 20